pkg/builders/default_storage_strategy.go

```diff
@@ -124,8 +124,8 @@ func (b DefaultStorageStrategy) GetAttrs(obj runtime.Object) (labels.Set, fields
 	}
 }
 
-func (b DefaultStorageStrategy) TriggerFunc(obj runtime.Object) []storage.MatchValue {
-	return []storage.MatchValue{}
+func (b DefaultStorageStrategy) TriggerFunc(obj runtime.Object) string {
+	return string
```

Comment:
storage.MatchValue is removed and use a plain string instead: https://github.com/kubernetes/kubernetes/pull/79793/files#diff-2a036e0adef795b5d716d44b8f412d7aL85